### PR TITLE
[AL-6155] [AL-6724] [AL-6738] Fix some test cases for ADV enabled accounts

### PR DESCRIPTION
**Pull request by attila-papai**
Changes included in this PR:
- marked the "TEXT" attachment type as deprecated, used "RAW_TEXT" instead
- added "is_adv_enabled" fixture to use in some test cases
- added "created_data_rows" property to "Task"

### CODE REVIEW

File: tests/integration/media/bulk_conversation.json [hunk 1]
```diff
@@ -1,113 +1,130 @@
...
-                    "userId": "Bot 002",
-                    "name": "Bot"
-                },
-                "align": "left",
-                "canLabel": false
-            }
-        ]
-    }
-]
+[{
```
Comment (attila-papai) [inline tests/integration/media/bulk_conversation.json:1]: this file was reformatted with `yapf`
the only meaningful changes are replacing "TEXT" with "RAW_TEXT"

File: labelbox/schema/task.py [hunk 2]
```diff
@@ -129,6 +129,16 @@ def failed_data_rows(self) -> Optional[Dict[str, Any]]:
         else:
             return None
 
+    @property
+    def created_data_rows(self) -> Optional[Dict[str, Any]]:
```
Comment (vbrodsky) [inline labelbox/schema/task.py:133]: Just for my information, this seems like a general `Task` class yet here we have methods specific to data row tasks like `failed_data_rows` and `created_data_rows` - a consequence of current task implementation?

File: tests/integration/conftest.py [hunk 3]
```diff
@@ -739,3 +739,10 @@ def run_dataset_export_v2_task(cls,
 @pytest.fixture
 def export_v2_test_helpers() -> Type[ExportV2Helpers]:
     return ExportV2Helpers()
+
+
+@pytest.fixture
+def is_adv_enabled(client) -> bool:
```
Comment (vbrodsky) [inline tests/integration/conftest.py:745]: can we make it session scope? PS Not sure whether it is possible, but would save on extra http traffic

File: labelbox/schema/task.py (hunk 2 above)
Comment (attila-papai) [reply labelbox/schema/task.py:133]: probably yes, each async task is represented by this Task class I guess, and some have these properties

File: tests/integration/conftest.py (hunk 3 above)
Comment (attila-papai) [reply tests/integration/conftest.py:745]: good idea, added!

**Review by vbrodsky — APPROVED** [review]

**Pull request merged**